experimental/regression_suite/shark-test-suite-models/sdxl/test_unet.py

```diff
@@ -104,11 +206,20 @@ def SDXL_UNET_COMMON_RUN_FLAGS(
     "--iree-codegen-llvmgpu-use-vector-distribution",
     "--iree-hip-waves-per-eu=2",
     "--iree-execution-model=async-external",
-    "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline,iree-preprocessing-pad-to-intrinsics)",
     "--iree-scheduling-dump-statistics-format=json",
     "--iree-scheduling-dump-statistics-file=compilation_info.json",
 ]
 
+FP16_UNET_FLAGS = [
+    f"--iree-codegen-transform-dialect-library={iree_test_path_extension}/attention_and_matmul_spec.mlir",
+    "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline,iree-preprocessing-pad-to-intrinsics)",
+]
+
+INT8_PUNET_FLAGS = [
+    f"--iree-codegen-transform-dialect-library={iree_test_path_extension}/attention_and_matmul_punet_spec.mlir",
+    "--iree-preprocessing-pass-pipeline=builtin.module(util.func(iree-global-opt-raise-special-ops, iree-flow-canonicalize), iree-preprocessing-transpose-convolution-pipeline, iree-preprocessing-pad-to-intrinsics, util.func(iree-preprocessing-generalize-linalg-matmul-experimental))",
+]
```

Comment:
How load-bearing are these preprocessing pass pipelines? Can we test with closer to default flags instead?

Reply:
I tried without the preprocessing pass pipeline at all and with the simpler flag being used for fp16. In both scenarios, it is failing accuracy :(

Reply:
We need to drive towards the compiler producing correct programs without a soup of flags and extra configuration files. If the test is failing, it's failing.

Reply:
Yeah, agreed. At the moment, the preprocess flag is needed for unet or the benchmark for unet is wildly off. For punet, it fails accuracy without its preprocess flag. With how much we care about these models, if we can have these testing running with some less than ideal flags, we can save tons of debugging time and hence spend more time on beating the green team in perf. Looks like we are slowly making steps toward the basic config with no longer needing spec files. Hopefully, we can get rid of this soon too

Reply:
Okay, fair enough. Thanks for checking. FYI @Groverkss (you recently filed some issues about making some other flags not required)